Sources/SwiftFormatWhitespaceLinter/WhitespaceLinter.swift

```diff
@@ -0,0 +1,332 @@
+// swift-tools-version:4.2
```

Comment:
It looks like this one line got accidentally copied from Package.swift.

Reply:
Done.